.claude/commands/amelia/coderabbit-review.md

```
@@ -0,0 +1,39 @@
+---
+description: fetch coderabbitai comments from current PR and evaluate with ka-eval-feedback
+---
+Fetch all comments from the `coderabbitai[bot]` user on the current PR and evaluate them.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)